v2/assistantv1/assistant_v1.go

```diff
@@ -3633,10 +3717,10 @@ func UnmarshalAgentAvailabilityMessage(m map[string]json.RawMessage, result inte
 // BulkClassifyOptions : The BulkClassify options.
 type BulkClassifyOptions struct {
 	// Unique identifier of the workspace.
-	WorkspaceID *string `json:"workspace_id" validate:"required,ne="`
```

Comment:
What is going on here?

Reply:
It's not removed, you can see the diff on line `3720`. For all options models parameters, the new generate code removes the `json` tags, which are used in Go to define how to marshal/unmarshal properties to/from JSON.

Within each function definition, you can see how the properties within the options model are handled. Specifically, we `nil` check each property and then based on the API definition it is appended to structs that become query parameters, form-data, or body parameters.